content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Dell EMC VMAX
+menuTitle: Dell EMC VMAX 
+description: "Dell EMC VMAX Support Matrix"
+weight: 20
+disableToc: false
+tags: ["user guide", "delfin", "drivers"]
+---
+
+### Model and Versions Supported
+
+|Models   | Firmware Versions   |
+|---------|---------------------|
+|         |                     |
+|         |                     |
+
+### Access information for register storage
+
+|Access Type   | Attributes   | Description |
```

Comment:
use mutiple rows in attributes  and descrition column for column for single access type , some thing as below


![image](https://user-images.githubusercontent.com/45681499/106747864-73b84000-664a-11eb-9dc5-81adaf531c26.png)

Reply:
Raised a issue #140

Reply:
This should be fixed after #142 is merged. 
@kumarashit @skdwriting Please merge #142 so that HTML support is enabled.

@Shruthi-1MN after #142 is merged you will be able to use the HTML table tags.